src/bin/pg_autoctl/pgsql.c

```diff
@@ -1048,9 +1050,29 @@ hostname_from_uri(const char *pguri, char *hostname, int maxHostLength)
 					return false;
 				}
 
-				break;
+				++found;
+			}
+		}
+
+		if (strcmp(option->keyword, "port") == 0)
+		{
+			if (option->val)
+			{
+				/* we expect a single port number in a monitor's URI */
+				*port = atoi(option->val);
+
```

Comment:
nit; whiteline is not adding much value IMHO

Reply:
Removed white line,